docs/_newsfragments/648.newandimproved.rst

```diff
@@ -0,0 +1,2 @@
+A new ``path`` converter capable of matching segments that include ``/``
```

Comment:
Would be nice to link to the new converter class from here, although not required to pass review.
For example, the RST code could read:
```rst
A new ``path`` :class:`converter <~falcon.routing.PathConverter>` capable of matching segments that include ``/``...
```

Reply:
will use that